testing/openjdk12/APKBUILD

```
@@ -0,0 +1,307 @@
+# Contributor: Simon Frankenberger <simon-alpine@fraho.eu>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=openjdk12
+pkgver=12.0.2_p10
+_pkgver=${pkgver/_p/+}
+pkgrel=0
+pkgdesc="Oracle OpenJDK 12"
+url="https://hg.openjdk.java.net/jdk-updates/jdk12u"
+arch="all !x86 !armhf !armv7" # openjdk11 is not yet available on 32 bit arches
+license="GPL-2.0 with Classpath"
+makedepends="autoconf
+bash
+gawk
+grep
+make
+openjdk11-jdk
+zip
+alsa-lib-dev
+cups-dev
+elfutils-dev
+fontconfig-dev
+freetype-dev
+giflib-dev
+lcms2-dev
+libexecinfo-dev
+libffi-dev
+libjpeg-turbo-dev
+libx11-dev
+libxext-dev
+libxrandr-dev
+libxrender-dev
+libxt-dev
+libxtst-dev
+linux-headers
+zlib-dev"
```

```suggestion
	zlib-dev
	"
```

Please indent lists.

---

done